docs/Makefile

```diff
@@ -25,6 +29,10 @@ help:
 livehtml:
 	@$(SPHINXAUTOBUILD) $(SPHINXAUTOBUILDOPTS) $(SPHINXOPTS) $(O) "$(SOURCEDIR)" "$(BUILDDIR)"
 
+docker-%:
```

Comment:
Will the `Makefile` work on windows? Or do we have to make the same rules on a `make.bat` file?

Reply:
me neither on batch-file coding, but regarding Docker on windows, the good news is that it works exactly the same.

I don't know how to translate the same rule on batch-file but if we had a simple `shell` rule like the following:

```Makefile
shell:
      cd $(SOURCEDIR)/.. && $(DOCKER) build -t $(DOCKERIMAGE) -f docs/Dockerfile .
      $(DOCKER) run -v $(shell get-proper-path):/usr/src/metalk8s --rm $(DOCKERIMAGE) --entrypoint=bash

# The equivalent in batch-file would be something like:

:shell
        %DOCKER% build -t %DOCKERIMAGE% -f Dockerfile ..
        %DOCKER% run -it  -v %~dp0:/usr/src/metalk8s --rm %DOCKERIMAGE% --entrypoint=bash
```
I know that it might not be as powerful that the `docker-%` rule but at least our windows user will get a proper environment to build their docs.

Reply:
I'd rather keep `tox` to be the entry-point to everything. When adding such `shell` rule, all of a sudden we have to explain how to build the docs using a second mechanism.

Reply:
Did not thought about that. That's a very good point. agree.